common/persistence/wrappers/metered/base.go

```diff
@@ -244,6 +268,44 @@ var emptyCountedMethods = map[string]struct {
 	},
 }
 
+var payloadSizeEmittingMethods = map[string]struct {
```

Comment:
Remark / feel free to ignore, I'm just wondering aloud: 

I kind of wish we didn't need to maintain such a map, but I feel like there's an opportunity for `List/Get/Start` wrapper structs to adhere to an interface like `MetricName() (Scope int, MetricName string)` in which they're capable of describing their scope themselves, rather than having them in one big centralized thing.